<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>572</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>935</width>
    <height>572</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>935</width>
    <height>572</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Многофункциональные часы</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/clock.ico</normaloff>../img/clock.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="baseSize">
       <size>
        <width>600</width>
        <height>800</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(65, 65, 65);
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Будильник </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>895</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>895</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 20pt &quot;Agency FB&quot;;
background-color: rgb(199, 199, 199);
color: rgb(43, 43, 43);
align-items: center;</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>true</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Имя</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Мелодия</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Значение</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="del_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 72pt &quot;Agency FB&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 72pt &quot;MS Serif&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Мировые часы  </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(231, 231, 231);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-radius : 10;
border : 2px solid black;</string>
          </property>
          <property name="text">
           <string>Выберите из списка город, время в котором хотите узнать.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QComboBox" name="choosebox">
          <property name="minimumSize">
           <size>
            <width>520</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>520</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>24</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(231, 231, 231);
font: 24pt &quot;MS Shell Dlg 2&quot;;
border-radius : 10;
border : 2px solid black;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(231, 231, 231);
font: 17pt &quot;MS Shell Dlg 2&quot;;
border-radius : 10;
border : 2px solid black;</string>
            </property>
            <property name="text">
             <string>Ваше текущее время:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLCDNumber" name="cur_lcd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(231, 231, 231);
font: 24pt &quot;MS Shell Dlg 2&quot;;
border-radius : 10;
border : 2px solid black;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLCDNumber" name="new_lcd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(231, 231, 231);
font: 24pt &quot;MS Shell Dlg 2&quot;;
border-radius : 10;
border : 2px solid black;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>450</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>876543</width>
              <height>876543</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(231, 231, 231);
font: 17pt &quot;MS Shell Dlg 2&quot;;
border-radius : 10;
border : 2px solid black;</string>
            </property>
            <property name="text">
             <string>Выбранное текущее время:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Agency FB&quot;;</string>
       </property>
       <attribute name="title">
        <string>Таймер </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSpinBox" name="hours">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(231, 231, 231);
border-radius : 15;
border : 2px solid black;</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="minutes">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border-radius : 15;
border : 2px solid black;</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="seconds">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border-radius : 15;
border : 2px solid black;</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="start_button">
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);
border-radius : 10;
border : 2px solid black;</string>
            </property>
            <property name="text">
             <string>Старт</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reset_button">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>22</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);
border-radius : 10;
border : 2px solid black;</string>
            </property>
            <property name="text">
             <string>Сброс</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup2</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QTextBrowser" name="taimer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>440</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>192</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 72pt &quot;Agency FB&quot;;
text-align: center;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);
border-radius : 10;
border : 2px solid black;
text-align: center;</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Agency FB'; font-size:72pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Agency FB</family>
              <pointsize>20</pointsize>
              <weight>9</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
font: 75 20pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Выбор мелодии для таймера:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(231, 231, 231);
font: 75 26pt &quot;Agency FB&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string> Секундомер</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(Clock/img/clock1.png);
border-radius : 70;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcd">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
font: 72pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
border-radius : 20;
border : 2px solid black;
text-align: center;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="stop_sec_button">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>24</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border-radius : 10;
border : 2px solid black;</string>
            </property>
            <property name="text">
             <string>Сброс</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup1</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="start_sec_button">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>24</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(52, 52, 52);
font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border-radius : 10;
border : 2px solid black;</string>
            </property>
            <property name="text">
             <string>Старт</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup1</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="flags">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>240</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(52, 52, 52);
color: rgb(231, 231, 231);
font: 10pt &quot;MS Shell Dlg 2&quot;;
border-radius : 20;
border : 2px solid black;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup1"/>
  <buttongroup name="buttonGroup2"/>
 </buttongroups>
</ui>
